### PR TITLE
 Issue #6966: aligned javadoc/xdoc for Translation

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6966 (part of #5750)

Second commit to ignore fields from inner `private` types.
This check has inner type `ResourceBundle` which also has a field named `baseName`.
Such fields should be ignored as we already do it for setters.

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java [hunk 1]
```diff
@@ -532,6 +524,19 @@ private static void visitClass(DetailAST ast) {
             }
         }
 
+        private static void visitField(DetailAST ast, DetailAST node) {
+            if (ScopeUtil.isInScope(node, Scope.PUBLIC)) {
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: I know the name came from the other method, but node is a bad name. It should have been named something like `parentAst`. I saw node and was thinking we were checking the modifier of the field, not the parent.

Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: `ast` named variables are not good names, even it used in 90% in our code. AST - abstract syntax tree. We have not whole tree but only some node of it.
Does it make sense to name `node` and `parentNode` ?

Comment (rnveach) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: I fine as long as we change it signify the parent in some way.

Comment (pbludov) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: Done. I'm not going to to rename all variables - it's a separate issue, but this method and its siblings changed.

**Pull request merged**